fhirstore/schema/search/search_methods.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import re
+import json
+
+from collections import defaultdict
+from elasticsearch import Elasticsearch
+
+
+def element_search(params):
```

Comment:
the function name could be improved, maybe `build_elastic_query()` or something like that ?

Reply:
I changed it to build_element_query